website/src/utils/PathsFactory.js

```diff
@@ -0,0 +1,47 @@
+export default class Pathsfactory{
+    #get(path, yaml) {
```

Comment:
`#get` what is this syntax?

Reply:
just like `_` in python means its private to a class `#` is the syntax to do it in js